docs/API_documentation/api_reference.md

```diff
@@ -1 +1,2 @@
-# API reference 
\ No newline at end of file
+# API reference
+<a href="{{coiote_site_link}}/apidoc/coiotedm/v3/redoc"> {{coiote_site_link}}/apidoc/coiotedm/v3/redoc </a>
```

Comment:
Actually every coiote installation serves redoc documentation under /apidoc/coiotedm/v3/redoc maybe I should include this information here? This matters because we have multiples coiote installations and versions may not always be the same.

Reply:
Only link was required in the ticket, but please let me know what you think